src/SimpleGraphsCore/SimpleGraphsCore.jl

```diff
@@ -0,0 +1,262 @@
+module SimpleGraphsCore
+
```

Comment:
This is the original core SimpleGraphs code + interface + constructors, stripped out into its own module that is used by the internal LightGraphs library code when it needs to construct graphs for whatever reason. Users should not be accessing this; the public facing bits are exported via the SimpleGraphs module.